policyengine_canada/parameters/gov/provinces/sk/benefits/afb/base.yaml

```
@@ -0,0 +1,9 @@
+description: Sasktachewan Active Family Benefit child base.
+values: 
+  2021-01-01: 150
+metadata:
+  unit: currency-CAD
+  label: Sasktachewan Active Family Benefit Child Base
+  reference: 
+  - title: Sasktachewan Active Families Benefit
```

Any luck finding a legal reference? If not could you add this comment.
```suggestion
  # Can't find a legal reference.
  - title: Sasktachewan Active Families Benefit
```